clic/src/openclbackend.cpp

```diff
@@ -997,7 +997,7 @@ static auto
 buildProgram(const Device::Pointer & device, const cl_program & program) -> void
 {
   auto   opencl_device = std::dynamic_pointer_cast<const OpenCLDevice>(device);
-  cl_int buildStatus = clBuildProgram(program, 1, &opencl_device->getCLDevice(), "-w", nullptr, nullptr);
+  cl_int buildStatus = clBuildProgram(program, 1, &opencl_device->getCLDevice(), "-w -cl-mad-enable", nullptr, nullptr);
```

Comment:
```suggestion
  cl_int buildStatus = clBuildProgram(program, 1, &opencl_device->getCLDevice(), "-w", nullptr, nullptr);
```
That was an unintended change